data/campaigns/Northern_Rebirth/scenarios/14a_Epilogue.cfg

```diff
@@ -131,15 +131,15 @@
         [/message]
         [message]
             id=Father Morvin
-            message= _ "This body would not claim sovereignty over any peoples or lands, but exist only to suppress warfare and raiding in the lands south and east of the Great Ocean, north of Wesnoth and west of the elvish forests."
+            message= _ "This body would not claim sovereignty over any peoples or lands, but exist to suppress warfare and raiding in the lands south and east of the Great Ocean, north of Wesnoth and west of the elvish forests."
```

Comment:
While I suppose what becomes the Northern Alliance does more than what Morvin suggests here, I think the point of the 'only' is to reinforce that the body Morvin proposes will not be a new faction staking its own claim to the lands of the North.

Reply:
Anyone else want to weigh in?

Reply:
You can leave the original then.

Reply:
Ok, did it.